staking/programs/integrity-pool/src/state/delegation_record.rs

```diff
@@ -7,11 +7,12 @@ use {
 #[account]
 #[derive(BorshSchema)]
 pub struct DelegationRecord {
-    pub last_epoch: u64,
+    pub last_epoch:             u64,
+    pub next_slash_event_index: u64,
 }
 
 impl DelegationRecord {
-    pub const LEN: usize = 8 + 8;
+    pub const LEN: usize = 1024;
```

Comment:
I'd keep this smaller since user is paying

Reply:
changed it to 128